airflow/kubernetes/pod_generator.py

```diff
@@ -62,6 +63,13 @@ def add_init_container(self,
     def _get_init_containers(self):
         return self.init_containers
 
+    def add_port(self, port):
```

Comment:
We're introducing typehints into the Airflow codebase.
```suggestion
    def add_port(self, port: int):
```

Reply:
Haven't had my coffee yet, I ment:
We're introducing typehints into the Airflow codebase.
```suggestion
    def add_port(self, port: Port):
```

Reply:
me neither, fixed this one and added type hinting for the other public functions